src/Compiler/CompilerOptions.cpp

```diff
@@ -363,6 +364,11 @@ static llvm::cl::opt<bool, true> useOnnxModelTypesOpt("useOnnxModelTypes",
     llvm::cl::location(useOnnxModelTypes), llvm::cl::init(true),
     llvm::cl::cat(OnnxMlirOptions));
 
+static llvm::cl::opt<bool, true> useOutputNameAsNodeNameOpt(
+    "useOutputNameAsNodeName", llvm::cl::desc("Use output name as node name."),
```

Comment:
maybe 
```suggestion
    "useOutputNameAsLocation", llvm::cl::desc("Use output name as location."),
```
is more precise (and also for the other places where we have the new option)

Reply:
Thanks, updated!